lib/timeline/Stack.js

```diff
@@ -125,8 +127,11 @@ export function stackSubgroups(items, margin, subgroups) {
       vertical: 0
     },
     true,
+    // eslint-disable-next-line no-unused-vars
     item => true,
+    // eslint-disable-next-line no-unused-vars
     item => true,
+    // eslint-disable-next-line no-unused-vars
     item => 0
```

Comment:
I would prefer if you removed the variable (`() => 0`) instead of suppressing rules